src/core/load-fragment-gperf.gperf.m4

```diff
@@ -166,15 +166,17 @@ $1.WatchdogSignal,               config_parse_signal,                0,
 )m4_dnl
 m4_define(`CGROUP_CONTEXT_CONFIG_ITEMS',
 `$1.Slice,                       config_parse_unit_slice,            0,                             0
+$1.AllowedCPUs,                  config_parse_allowed_cpus,          0,                             offsetof($1, cgroup_context)
+$1.AllowedMemoryNodes,           config_parse_allowed_mems,          0,                             offsetof($1, cgroup_context)
+$1.CPUSetCpus,                   config_parse_allowed_cpus,          0,                             offsetof($1, cgroup_context)
+$1.CPUSetMems,                   config_parse_allowed_mems,          0,                             offsetof($1, cgroup_context)
 $1.CPUAccounting,                config_parse_bool,                  0,                             offsetof($1, cgroup_context.cpu_accounting)
 $1.CPUWeight,                    config_parse_cg_weight,             0,                             offsetof($1, cgroup_context.cpu_weight)
 $1.StartupCPUWeight,             config_parse_cg_weight,             0,                             offsetof($1, cgroup_context.startup_cpu_weight)
 $1.CPUShares,                    config_parse_cpu_shares,            0,                             offsetof($1, cgroup_context.cpu_shares)
 $1.StartupCPUShares,             config_parse_cpu_shares,            0,                             offsetof($1, cgroup_context.startup_cpu_shares)
 $1.CPUQuota,                     config_parse_cpu_quota,             0,                             offsetof($1, cgroup_context)
 $1.CPUQuotaPeriodSec,            config_parse_sec_def_infinity,      0,                             offsetof($1, cgroup_context.cpu_quota_period_usec)
-$1.CPUSetCpus,                   config_parse_cpuset_cpus,           0,                             offsetof($1, cgroup_context)
-$1.CPUSetMems,                   config_parse_cpuset_mems,           0,                             offsetof($1, cgroup_context)
```

Comment:
na, let's drop this props. rc1 should give us the ability to drop this again. I mean, it's unlikely people immediately used that...

(also, we genreally hide props we don#t want to support anymore usin SD_BUS_VTABLE_HIDDEN, and we add a comment about it ideally).

But really, let's just kill this. rc1 is not a stable version there's no stability guaranteed. i mean, by all means tag another rc the instant is merged, if you want to "erase" the mistake quickly, but let's not collect debt from rc "releases" like this

Reply:
OK. Let's assume nobody used this.